### ISSUE TITLE
Oversize folders aren't displaying in Storage Notes.

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pulfalight#910]
This is because our magic storage note is only looking at boxes. In the case of oversize folders there is no box.

https://findingaids.princeton.edu/catalog/MC120_c41715-74796 should show that it's stored in Mudd in the storage note.

https://findingaids.princeton.edu/catalog/MC120 should say something like

"ReCAP (rcpph): Box 1-665, Mudd (mudd): Oversize Folder 1 (cabinet 9 drawer 1)"

**Comment by phoebeno** [comment]
Here is a storage note that is missing its container@type="volume"s
[](https://findingaids.princeton.edu/catalog/AC120) . . . but actually, they would be dreadful if they showed up in the storage note because some of them are not numerical, like this: "Volume September/November 2002"

Here is a storage note that is missing its container@type="item"s: [](https://findingaids.princeton.edu/catalog/AC053)
(it contains items 1-21). I can't think of a case where it is really a priority to display these items at the moment; the real issue is when most things in a collection are at ReCAP but the material left at another repository is not a "box"; it appears that there may be no material at Mudd when there is.

**Comment by tpendragon** [comment]
@phoebeno Rather than splitting things out, do we need "ReCAP (rcpph): Box 1-665, Mudd (mudd): Some non-box content." or something?

**Comment by phoebeno** [comment]
I love that idea. I mean, in most collections with volumes, I think it will be more straightforward and would just look like "Volume 1-23". Items also. So people might miss the chance to be more specific there. But the oversize folders are a little ridiculous. Would it be possible to have it just say "oversize folders" if that container type is present? 
My main goal right now is simply to have "mudd" appear. But I can imagine people wanting to know WHICH volumes might be at ReCAP and which volumes here, at some point. 
In fact, all the problem volumes in AC120 are in the location "secure cage." Is that going to show up?

**Comment by tpendragon** [comment]
@phoebeno I'm not sure what the secure cage shows up as. What's the location code?

Maybe as a first pass we just do something like "ReCAP (rcpph): Box 1-665, Mudd (mudd): Oversize folders", by just making any non-box types just display the container types that are there rather than the associated numbers.

**Comment by phoebeno** [comment]
Yes, some volumes are here on shelves, like a library! This solution sounds good to me. Thanks!

**Comment by phoebeno** [comment]
Oh, I didn't see the secure cage question. It gets this: container altrender="sc"

**Comment by tpendragon** [comment]
@phoebeno I don't think those will show up - presumably they should show up as "Mudd"?

**Comment by phoebeno** [comment]
Can we leave them off for now?

**Comment by tpendragon** [comment]
@phoebeno Yup, that's zero work, which is the best amount. 😀
